pkg/components/prometheus-operator/template.go

```diff
@@ -56,6 +57,22 @@ grafana:
       searchNamespace: ALL
   rbac:
     pspUseAppArmor: false
+
+  ingress:
+    # If user does not provide any ingress host value then following will be 'false', thus disabling
+    # the ingress section.
+    enabled: {{printf "%t" (ne .GrafanaIngressHost "")}}
+    annotations:
+      kubernetes.io/ingress.class: contour
+      kubernetes.io/tls-acme: "true"
+      cert-manager.io/cluster-issuer: "letsencrypt-production"
```

Comment:
Should we parameterize this too like we did in #437?

Reply:
That's what I have asked in https://github.com/kinvolk/lokomotive/pull/468#discussion_r430918969